tools/mc_ledger_test.py

```diff
@@ -0,0 +1,93 @@
+# Copyright (c) 2022 MobileCoin, Inc.
+import local_network
+import time
+
+from test_utils.utilities import parse_network_type_cmd_line_args
+from test_utils import fullservice as fslib
+
+# Shoving the raw request into a python class would allow us to just access these things as members of the class
+# TODO: create a python dataclass to hold these
+balance = 'balance'
+pending = 'pending_pmob'
+unspent = 'unspent_pmob'
+spent = 'spent_pmob'
+fee = 'fee_pmob'
```

Comment:
Nit: I don't think this is making the code any more readable, but I think if you are going to use these global constants, it would be better if they are uppercase.